scripts/addon/build.py

```diff
@@ -438,6 +438,11 @@ def qtquery(qmake, propname):
                 xlifftool_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "..", "utils", "xlifftool.py")
                 xlifftool_cmd = [sys.executable, xlifftool_path, "-C", f"--locale={locale}", xliff_path]
                 xlifftool = subprocess.run(xlifftool_cmd, capture_output=True)
+                # This completeness metric can be out-of-date, sometimes reporting a higher-than-actual completeness after a new string is
+                # added to the app. This happens because the .xliff file doesn't have all the source strings, so an incorrect denomenator is used.
```

Comment:
```suggestion
                # added to the app. This happens because the .xliff file doesn't have all the source strings, so an incorrect denominator is used.
```